Validation/GEMCosmicMuonStand/test/StandGeometryConfiguration_run1.csv

```diff
@@ -1,15 +1,15 @@
 RunNumber,1,

```

Comment:
i would rather these settings goto /data dir

Reply:
Do you mean to place the gem11L_c1_r1.xml and other xml files in the folder gemcrs/src/Geometry/MuonCommonData/data/ ?

Reply:
Validation/GEMCosmicMuonStand/data